write-fonts/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["text-processing", "parsing", "graphics"]
 [dependencies]
 font-types = { version = "0.1.5", path = "../font-types" }
 read-fonts = { version = "0.2.0", path = "../read-fonts" }
+log = "0.4"
```

Comment:
I wonder, do we actually unconditionally want log? Suppose we had a subsetter for use in pdf processing, should log be required?

Reply:
This will also come up in #387, where I add a dependency to write graphviz files. I could put both of these behind a 'diagnostics' feature, if desired?